### DISCUSSION TITLE
More prominent link to documentation

### DISCUSSION THREAD

**Discussion by samuell** [cozodb/cozo#4] [Ideas]
Hi, thanks for very interesting work!

A small head up: It is a bit hard to find the [documentation](https://cozodb.github.io/current/manual/) (I wouldn't have found it if I didn't see it mentioned in a HN thread). Maybe add the link as the official URL on the repo, or as a shield/badge in the README?

EDIT: Also, an easily findable link to the [tutorial](https://nbviewer.org/github/cozodb/cozo-docs/blob/main/tutorial/tutorial.ipynb) would be great! (Just found out about it :) )

**Comment by zh217** [comment]
Done, thank you for the suggestion!
